scene/3d/area_3d.cpp

```diff
@@ -105,6 +105,52 @@ real_t Area3D::get_priority() const {
 	return priority;
 }
 
+void Area3D::set_wind_force_magnitude(real_t p_wind_force_magnitude) {
+	wind_force_magnitude = p_wind_force_magnitude;
+	if (is_inside_tree()) {
+		_initialize_wind();
+	}
+}
+
+real_t Area3D::get_wind_force_magnitude() const {
+	return wind_force_magnitude;
+}
+
+void Area3D::set_wind_attenuation_factor(real_t p_wind_force_attenuation_factor) {
+	wind_attenuation_factor = p_wind_force_attenuation_factor;
```

Comment:
This should also have an error on negative values, for the case where it's called by script.

Reply:
I added in error the call that area_3d makes to area_3d_sw to set the param, which should catch this